.cirrus.yml

```diff
@@ -201,6 +201,15 @@ centos7_task:
   << : *CI_TEMPLATE
   << : *SKIP_TASK_ON_PR
 
+debian_unstable_task:
```

Comment:
In retrospect, may seem curious we run unstable but not testing, but good for now I think :-)